### PR TITLE
Add help page for banana penalties

### LINKED ISSUE #2174: Add banana penalties into in-game help
I humbly request to extend the in-game help with a page explaining all the bad things which can happen to you when you drive over a banana, i.e. anchor, parachute, bomb.

### PR DESCRIPTION
**Pull request by LoadingPleaseWait**
This closes #2174. More information could still be added and the GUI could also be improved to look better.

### CODE REVIEW

File: src/states_screens/help_screen_5.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
+//  SuperTuxKart - a fun racing game with go-kart
+//  Copyright (C) 2009-2015 Marianne Gagnon
```
Comment (qwertychouskie) [inline src/states_screens/help_screen_5.cpp]: These should be "Copyright (C) 2000-2016 The SuperTuxKart team and contributers".

File: src/states_screens/help_screen_5.hpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
+//  SuperTuxKart - a fun racing game with go-kart
+//  Copyright (C) 2009-2015 Marianne Gagnon
```
Comment (qwertychouskie) [inline src/states_screens/help_screen_5.hpp]: Same here.

**Comment by auriamg** [comment]
Hi, this is a good start, but it would be better if there were icons, and the text for the bomb should mention that it can be passed to another kart

File: src/states_screens/help_screen_5.cpp (hunk 1 above)
Comment (Benau) [inline src/states_screens/help_screen_5.cpp]: Use UNIFORM HEADING as in from other cpp files (something like Copyright (C) xxxx SuperTuxKart-Team)

**Comment by LoadingPleaseWait** [comment]
Thank you for your responses. I'll change the copyright comment, try and add icons and improve the text.

File: src/states_screens/help_screen_5.cpp (hunk 1 above)
Comment (qwertychouskie) [inline src/states_screens/help_screen_5.cpp]: Better grammar would be "the SuperTuxKart Team".  Not a huge deal, but a bit nicer to look at.

**Comment by Benau** [comment]
qwertychouskie you are VERY annoying by keep suggesting useless thing

**Comment by LoadingPleaseWait** [comment]
I have amended my commit so that there are icons in the help page and there is more information about the bomb.

**Pull request merged**